workflow/workflow.go

```
@@ -299,6 +299,9 @@ func NewPublish(projectInfo ProjectInfo, fs afero.Fs) (Workflow, error) {
 
 	if chartDirectoryExists {
 		for _, c := range projectInfo.Channels {
+			if c == "" {
+				continue
+			}
```

Why would we have empty channel names? Isn't that input parameters validation issue?

---

> Why would we have empty channel names?

If we pass something like `--channels beta,,stable`

> Isn't that input parameters validation issue?

It can be seen like that, currently we don't have actual input parameters validation logic. Do you think we should add it and fail if we pass empty channels?

---

I think we should just validate the input and fail in case we get something like `--channels beta,,stable`. Input like this indicates for me that something is wrong and we should be aware of it.

---

I feel the same. I'm just afraid something will fail silently in the future.

---

Makes sense, I've updated it to fail on empty channels received, PTAL.